modules/exploits/unix/http/syncovery_linux_rce_2022_36534.rb

```diff
@@ -0,0 +1,194 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Syncovery For Linux Web-GUI Authenticated Remote Command Execution',
+        'Description' => %q{
+          This module exploits an authenticated command injection vulnerability in the Web GUI of Syncovery File Sync & Backup Software for Linux.
+          Successful exploitation results in remote code execution under the context of the root user.
+
+          Syncovery allows an authenticated user to create jobs, which are executed before/after a profile is run.
+          Jobs can contain arbitrary system commands and will be executed as root.
+          A valid username and password or a session token is needed to exploit the vulnerability.
+          The profile will be deleted afterwards to disguise the attack. IOC_IN_LOGS are present, but can be deleted manually.
```

Comment:
Can we add a print_status or similar to the user, so they know to remove IOC manually if a session has been opened

Reply:
Sure. It is also possible to clear all logs with one request. But I dont know if this would be too intrusive.

Reply:
Done.